app/assets/javascripts/discourse/app/components/user-menu/default-reviewable-item.hbs

```diff
@@ -0,0 +1,9 @@
+<li class={{unless this.reviewable.pending "reviewed"}}>
+  <LinkTo @route="review.show" @model={{this.reviewable.id}}>
+    {{d-icon this.icon}}
+    <div>
```

Comment:
do we need this div ?

Reply:
yeah unfortunately there are some CSS roles that need this div (I'm re-using the existing styles of the old notifications menu for the new menu including the reviewables tab so I can't change the styles until we kill the old menu).